Date_Holidays offers two scripts that hardcode the path to PHP.

Corresponding PEAR bug:

https://pear.php.net/bugs/bug.php?id=16423

diff -Naur pear-dh-compile-translationfile pear-dh-compile-translationfile
--- a/a/a/pear-dh-compile-translationfile	2009-03-24 01:18:21.000000000 +0100
+++ a/a/a/pear-dh-compile-translationfile	2009-07-10 08:07:51.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/php
+#!@php_bin@
 
 <?php
 /**
diff -Naur pear-dh-ini2xml pear-dh-ini2xml
--- a/a/a/pear-dh-ini2xml	2009-03-24 01:18:21.000000000 +0100
+++ a/a/a/pear-dh-ini2xml	2009-07-10 08:07:47.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/php
+#!@php_bin@
 <?php
 /**
  * Converts the old-style INI-language files into XML-files that
@@ -58,4 +58,4 @@
 
 
 
-?>
\ Kein Zeilenumbruch am Dateiende.
+?>
diff -Naur package.xml package.xml
--- a/a/a/package.xml	2009-03-24 01:18:21.000000000 +0100
+++ a/a/a/package.xml	2009-07-10 08:07:18.000000000 +0200
@@ -87,8 +87,12 @@
    <file baseinstalldir="Date"  name="tests/test_getHolidays.php" role="test" />
    <file baseinstalldir="Date"  name="tests/test_missingLocale.php" role="test" />
    <file baseinstalldir="Date"  name="Holidays.php" role="php" />
-   <file baseinstalldir="/"  name="pear-dh-compile-translationfile" role="script" />
-   <file baseinstalldir="/"  name="pear-dh-ini2xml" role="script" />
+   <file baseinstalldir="/"  name="pear-dh-compile-translationfile" role="script">
+     <replace from="@php_bin@" to="php_bin" type="pear-config"/>
+   </file>
+   <file baseinstalldir="/"  name="pear-dh-ini2xml" role="script">
+     <replace from="@php_bin@" to="php_bin" type="pear-config"/>
+   </file>
   </dir>
  </contents>
  <dependencies>
